src/pages/HistoryDetail.vue

```diff
@@ -0,0 +1,13 @@
+<script>
+import { connect } from 'vuex-connect'
+import HistoryDetail from '@/components/History/HistoryDetail'
+
+export default connect({
+  gettersToProps: {
+    entry: 'history/active',
+  },
+  lifecycle: {
+    mounted: ({ dispatch, getters }) => dispatch('history/setActive', getters['route/params'].historyId),
```

Comment:
I guess this has the same non-reactive problem as the history list did before? If the route params change, this will not update... how does it avoid it?

Reply:
It does not work. I added a new commit to work on it, but it doesn't work and I don't know how to solve it. Maybe I'd have to get rid of vuex-connect.

Reply:
Yup, that's what had to happen with the history stuff. I think getting rid of vuex-connect might be the only way :/ (because then you can add watchers using a normal component, which vuex-connect could let you do in theory... but does not seem officially supported so would be unwise to hack in imo).

Reply:
I fixed it by putting the loading into the router vuex plugin, which seems our currently best solution for handling this.
(I also noticed that the history list view shows sometimes wrong entries when using back navigation, need to investigate more)